### ISSUE TITLE
No distributions at all for reverse_geocoding

### ISSUE THREAD

**Issue by jeremydunn-wtw** [thampiman/reverse-geocoder#45]
Does this pip install work ? Doesn't seem to find the library. Is it still available ? 

![image](https://user-images.githubusercontent.com/31994705/53962907-11640700-40e4-11e9-8f98-2b9ac77d7a1e.png)

Seems to just time out. 

Regards. 

Jeremy.

**Comment by MorganRodgers** [comment]
@jsdunn50 yes `pip install` works. Your error which was a [ConnectionTimeoutError](http://docs.python-requests.org/en/master/api/#requests.ConnectTimeout) is a network error thrown by Requests in Pip and is not related to this project.
